docs/guide/platform_environment.md

```diff
@@ -114,6 +114,89 @@ For this reason, if you intend to use this in a production application, you shou
 For more information on Node.js, see this guide.
 
 
+#### WASM backend
+
+TensorFlow.js provides a [WebAssembly backend](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-wasm/README.md) (`wasm`), which offers CPU acceleration and can be used as an alternative to the vanilla JavaScript CPU (`cpu`) and WebGL accelerated (`webgl`) backends.  To use it:
+
+```js
+// Set the backend to WASM and wait for the module to be ready.
+tf.setBackend('wasm');
+tf.ready().then(() => {...});
+```
+
+If your server is serving the `.wasm` file on a different path or a different name, use `setWasmPath` before you initialize the backend. See the ["Using Bundlers"](https://github.com/tensorflow/tfjs/tree/master/tfjs-backend-wasm#using-bundlers) section in the README for more info:
+
+```js
+import {setWasmPath} from '@tensorflow/tfjs-backend-wasm';
+setWasmPath(yourCustomPath);
+tf.setBackend('wasm');
+tf.ready().then(() => {...});
+```
+
+> Note: TensorFlow.js defines a priority for each backend and will automatically choose the best supported backend for a given environment. To explicitly use the WASM backend, we need to call `tf.setBackend('wasm')`.
+
+##### Why WASM?
+
+[WASM](https://webassembly.org/) was introduced in 2015 as a new web-based binary format, providing programs written
+in JavaScript, C, C++, etc. a compilation target for running on the web. WASM has been [supported](https://webassembly.org/roadmap/)
+by Chrome, Safari, Firefox, and Edge since 2017, and is supported by [90% of devices](https://caniuse.com/#feat=wasm)
+worldwide.
+
+**Performance**
+
+*Versus JavaScript*: WASM binaries are generally much faster than JavaScript bundles for browsers to load, parse, and
+execute. JavaScript is dynamically typed and garbage collected, which can cause slowdowns at runtime.
+
+*Versus WebGL*: WebGL is faster than WASM for most models, but for tiny models WASM can outperform WebGL due to the
+fixed overhead costs of executing WebGL shaders. The “When should I use WASM” section below discusses heuristics for
+making this decision.
+
+**Portability and Stability**
+
+WASM has portable 32-bit float arithmetic, offering precision parity across all devices. WebGL, on the other hand, is
+hardware-specific and different devices can have varying precision (e.g. fallback to 16-bit floats on iOS devices).
+
+Like WebGL, WASM is officially supported by all major browsers.
+
+##### When should I use WASM?
+
+**Model size and computational demand**
+
+In general, WASM is a good choice when models are smaller or you care about older mobile devices that lack WebGL
```

Comment:
Not just older mobile devices, low-end devices often have less powerful GPUs than CPUs (e.g. Figure 4 in [this paper](https://research.fb.com/wp-content/uploads/2018/12/Machine-Learning-at-Facebook-Understanding-Inference-at-the-Edge.pdf))